helm/.dagger/main.go

```diff
@@ -28,34 +38,37 @@ func (h *Helm) Lint(ctx context.Context) error {
 	return err
 }
 
+// Test the helm chart on an ephemeral K3S service
 func (h *Helm) Test(ctx context.Context) error {
-	cli := dag.DaggerCli().Binary()
 	k3s := dag.K3S("helm-test")
-
 	// NOTE: force starting here - without this, the config won't be generated
 	k3ssvc, err := k3s.Server().Start(ctx)
 	if err != nil {
 		return err
 	}
-
 	test, err := h.chart().
-		WithMountedFile("/usr/bin/dagger", cli).
+		WithMountedFile("/usr/bin/dagger", dag.DaggerCli().Binary()).
 		WithServiceBinding("helm-test", k3ssvc).
 		WithFile("/.kube/config", k3s.Config()).
 		WithEnvVariable("KUBECONFIG", "/.kube/config").
 		WithEnvVariable("CACHEBUSTER", identity.NewID()).
 		WithExec([]string{"kubectl", "get", "nodes"}).
-		WithExec([]string{"helm", "install", "--wait", "--create-namespace", "--namespace=dagger", "--set=engine.image.ref=registry.dagger.io/engine:main", "dagger", "."}).
+		WithExec([]string{"helm", "install", "--wait", "--create-namespace", "--namespace=dagger", "dagger", "."}).
```

Comment:
Setting this image is deliberate, we should keep it as is:
- https://github.com/dagger/dagger/pull/8305

Reply:
also done by error, let me clean up real quick